man/use_build_ignore.Rd

```diff
@@ -26,21 +26,9 @@ turn a filename into a regular expression that will only match that
 path. Repeated entries will be silently removed.
 }
 \seealso{
-Other infrastructure: \code{\link{add_rstudio_project}},
-  \code{\link{add_test_infrastructure}},
-  \code{\link{add_test_infrastructure}},
-  \code{\link{add_travis}}, \code{\link{add_travis}},
-  \code{\link{add_travis}}, \code{\link{add_travis}},
-  \code{\link{infrastructure}}, \code{\link{use_appveyor}},
-  \code{\link{use_code_of_conduct}},
-  \code{\link{use_coverage}}, \code{\link{use_cran_badge}},
-  \code{\link{use_cran_comments}},
-  \code{\link{use_package_doc}}, \code{\link{use_rcpp}},
-  \code{\link{use_revdep}}, \code{\link{use_rstudio}},
-  \code{\link{use_test}}, \code{\link{use_testthat}},
-  \code{\link{use_travis}}, \code{\link{use_vignette}};
-  \code{\link{use_data_raw}}; \code{\link{use_data}};
-  \code{\link{use_package}}; \code{\link{use_readme_rmd}}
+Other infrastructure: \code{\link{infrastructure}},
```

Comment:
That's new roxygen2 5.0.0 behaviour - it just links to the main entry, not every alias.  

I just pushed a commit that upgrades to roxygen2 5.0.0. Could you please merge/rebase?